snap/local/register-device.sh

```diff
@@ -7,6 +7,19 @@ if [ ! -f "$CONFIGURATION_FILE_PATH" ]; then
     exit 1
 fi
 
-$SNAP/bin/cos-registration-agent --shared-data-path $SNAP_COMMON/rob-cos-shared-data --grafana-dashboards $SNAP_COMMON/configuration/grafana_dashboards --foxglove-studio-dashboards $SNAP_COMMON/configuration/foxglove_layouts setup -c $CONFIGURATION_FILE_PATH
+REGISTRATION_CMD="$SNAP/bin/cos-registration-agent --shared-data-path $SNAP_COMMON/rob-cos-shared-data"
+
+# Check if grafan_dashboards directory exists in configuration
+if [ -d "$SNAP_COMMON/configuration/grafana_dashboards" ]; then
```

Comment:
Use curly brackets.

Reply:
Fixed in ca5b068c61962194020a8e925d01b859e6e2ef40